boards/renesas/ek_ra6m3/ek_ra6m3.dts

```diff
@@ -60,6 +60,17 @@
 	};
 };
 
+&iic2 {
+	status = "okay";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	interrupts = <91 1>, <92 1>, <93 1>, <94 1>;
```

Comment:
can you please tell me where these interrupt numbers come from and how they are defined?

Reply:
For the interrupt source, you can refer to '36.14 Interrupt Sources - RA6M3 MCU Group HWM'. For the interrupt line mapping, you can refer to '14. Interrupt Controller Unit (ICU) - RA6M3 MCU Group HWM' and '14.3 Vector Table - RA6M3 MCU Group HWM'.
Btw, I saw a thread discussing how Renesas RA MCU maps interrupts here https://discord.com/channels/720317445772017664/941333129556279366/1303710804671201342